pytorch/pytorch-native/src/main/native/ai_djl_pytorch_jni_PyTorchLibrary_inference.cc

```diff
@@ -99,3 +168,40 @@ JNIEXPORT void JNICALL Java_ai_djl_pytorch_jni_PyTorchLibrary_torchDeleteModule(
   delete module_ptr;
   API_END()
 }
+
+JNIEXPORT void JNICALL Java_ai_djl_pytorch_jni_PyTorchLibrary_moduleSave(
+        JNIEnv* env, jobject jthis, jlong jhandle, jstring jpath) {
+    API_BEGIN()
+    auto* module_ptr = reinterpret_cast<torch::jit::script::Module*>(jhandle);
+    module_ptr->save(djl::utils::jni::GetStringFromJString(env, jpath));
+    API_END()
+}
+
+JNIEXPORT jlongArray JNICALL Java_ai_djl_pytorch_jni_PyTorchLibrary_moduleGetParams(
+        JNIEnv* env, jobject jthis, jlong jhandle) {
+    API_BEGIN()
+    auto* module_ptr = reinterpret_cast<torch::jit::script::Module*>(jhandle);
+    std::vector<jlong> jptrs;
+    for (const auto tensor : module_ptr->parameters())
```

Comment:
format c code

Reply:
tensor is considered as big object, will reference here
```suggestion
    for (const auto& tensor : module_ptr->parameters())
```